components/omega/configs/Default.yml

```diff
@@ -46,7 +46,8 @@ Omega:
       FreqUnits: OnStartup
       UseStartEnd: false
       Contents:
-        - Restart
+        - State
+        - Base
```

Comment:
Should we add `State` and `AuxiliaryState` to the `History` IOStream here?

Reply:
Yes, I think that gives Xylar the SSH he needs? I'll add that.

Reply:
Actually, AuxState does not have an SSH field. I can certainly add State to the history.

Reply:
Oh wait, I found it. Sorry...

Reply:
@philipwjones - if you're able to push this change, I can restest quick and approve.